src/Mastodon.elm

```diff
@@ -502,3 +517,31 @@ postStatus client statusRequestBody =
         |> HttpBuilder.withHeader "Authorization" ("Bearer " ++ client.token)
         |> HttpBuilder.withExpect (Http.expectJson statusDecoder)
         |> HttpBuilder.withJsonBody (statusRequestBodyEncoder statusRequestBody)
+
+
+reblog : Client -> Int -> Request Status
+reblog client id =
+    HttpBuilder.post (client.server ++ "/api/v1/statuses/" ++ (toString id) ++ "/reblog")
```

Comment:
I think that at some point, we should add an unique reference for all these urls. Just a `ApiUrl.elm` file containing variables and functions to build the url in a typesafe way should be fine. I'll fill an issue to discuss it.